src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -5,15 +5,34 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
+@Table(name = "carts")
 public class Cart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "cart_id")
+    @NotNull
     private Long id;
+    //private User user;
+
+
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "join_products_cart",
+            joinColumns = {@JoinColumn(name = "cart_id", referencedColumnName = "cart_id")},
+            inverseJoinColumns = {@JoinColumn(name = "product_id", referencedColumnName = "product_id")}
+    )
     private List<Product> products = new ArrayList<>();
-    private User user;
```

Comment:
Wysypuje tetsy na CircleCi, może przez brak pola User user;

Reply:
jak zostawiam, to mam komunikat 'Basic' attribute type should not be 'Persistence Entity' i wtedy to mi projektu nie buduje i się sypią testy

Reply:
hmmm

Reply:
a może przez brak:
@AllArgsConstructor
@NoArgsConstructor
w domain/Product? 
W ostatnim commicie tego nie ma. Wtedy w testach konstruktora brak

Reply:
jak rozwiniesz commita, to tam są te adnotacje